tirgei/Questioner-API
# #163111722 Refactored endpoints to include JWT

#### What does this PR do?
This Pull Request fixes endpoints to include JWT

#### Description of Task to be completed?
The tasks for this PR is:
- Add JWT required to endpoints that require authorization

#### How should this be manually tested?
Clone this repository on your machine:
```$ git clone https://github.com/tirgei/Questioner-API.git ```

CD into the project folder on your machine
```$ cd Questioner-API ```

Check out the branch for this PR
``` $ git checkout bg-fix-jwt-163111722```


#### Pivotal Tracker story associated with this PR
[#163111722](https://www.pivotaltracker.com/story/show/163111722)
